CMakeExt/MPI.cmake

```diff
@@ -30,9 +30,18 @@ if (MPI_INCLUDE_PATH AND MPI_LIBRARY)
   elseif ("${MPI_INCLUDE_PATH}" MATCHES "openmpi")
```

Comment:
Implemented. Unfortunately, Cray MPI claims to be MPICH so we still need to check the include path.